static/src/javascripts/projects/common/modules/experiments/tests/february-moment-banner.js

```diff
@@ -15,6 +16,15 @@ const thankYouCopy =
 
 const campaignId = 'empower_campaign';
 
+// These test params must be set on engagementBannerParams *and* passed into canShowBannerSync
```

Comment:
@johnduffell I've just pushed changes to this file after rebasing with your changes